docs/intro/tutorial07.txt

```diff
@@ -418,6 +418,11 @@ uses a template variable called ``app_list``. That variable contains every
 installed Django app. Instead of using that, you can hard-code links to
 object-specific admin pages in whatever way you think is best.
 
+
+
```

Comment:
```suggestion
```

Reply:
This feels like something that should be handled in pre-commit.